src/main/groovy/org/shipkit/internal/config/GitOriginRepoProvider.java

```diff
@@ -0,0 +1,32 @@
+package org.shipkit.internal.config;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.shipkit.internal.exec.ProcessRunner;
+
+public class GitOriginRepoProvider {
+
+    private static final Logger LOG = Logging.getLogger(GitOriginRepoProvider.class);
+    public static final String FALLBACK_REPO = "mockito/mockito-release-tools-example";
```

Comment:
Nitpick: let's avoid hardcoding the fallback repo in this class. Let's make this class responsible for identifying the origin repo but not to make a decision what to do when that operation fails.

I suggest to pass fallback repo as String to the 'getOriginGitRepo' method. This way, the client decides what repo to use, in case obtaining the repo fails.

How does it sound?